ftl/python/testdata/packages_test/structure_test.yaml

```diff
@@ -16,3 +16,9 @@ fileExistenceTests:
   path: '/env/lib/python2.7/site-packages/flask'
   shouldExist: true
   isDirectory: false
+
+commandTests:
+- name: 'venv setup correctly'
+  command: 'cat'
+  args: ['/env/bin/activate']
+  expectedOutput: ['"/env"']
```

Comment:
need the double quotes here?

Reply:
yes, /env would be in the suffix of the incorrect path that this fixes